kraken/chaos_recommender/prometheus.py

```diff
@@ -15,14 +14,6 @@ def convert_data_to_dataframe(data, label):
     return df
 
 
-def get_value_from_data(data, service):
```

Comment:
@mudverma removed this method that was unused and was containing a typo

Reply:
Agreed